src/Symfony/Component/AssetMapper/ImportMap/ImportMapRenderer.php

```diff
@@ -21,34 +23,58 @@ class ImportMapRenderer
 {
     public function __construct(
         private readonly ImportMapManager $importMapManager,
+        private readonly ?Packages $assetPackages = null,
         private readonly string $charset = 'UTF-8',
         private readonly string|false $polyfillUrl = ImportMapManager::POLYFILL_URL,
         private readonly array $scriptAttributes = [],
     ) {
     }
 
-    public function render(string $entryPoint = null, array $attributes = []): string
+    public function render(string|array $entryPointNames, array $attributes = []): string
```

Comment:
like in ImportMapRuntime?
```suggestion
    public function render(string|array $entryPoint, array $attributes = []): string
```